scripts/twister

```diff
@@ -685,6 +686,10 @@ structure in the main Zephyr tree: boards/<arch>/<board_name>/""")
         help="Execute time-consuming test cases that have been marked "
              "as 'slow' in testcase.yaml. Normally these are only built.")
 
+    parser.add_argument(
+        "--seed", type=int, default=random.randrange(8192),
+        help="Seed for pseudo-random number generator")
```

Comment:
If this is only used in the native targets, it should say that it is so, and only affecting the native target random generator. Otherwise this would seem to be some twister pseudo-random number generator.

Reply:
Sure, I can update and mention that this is used for native posix only.